doc/headset.md

```diff
@@ -78,6 +79,11 @@ sdk.headset.headsetEvents.subscribe(value: {
 
 ```
 
+### Properties
+
+#### `currentSelectedImplementation`
+The currently selected implementation from the headset library. This will help with checking proper flags for various functionality such as `this.currentSelectedImplementation.isConnected`
```

Comment:
This comment is kind of odd. It doesn't seem like consumers of the sdk need to know about the `this.currentSelectedImplementation.isConnected` logic behind the scenes. I think just letting them know what this property is would be sufficient. You could either leave it as just: 
> _The currently selected implementation from the headset library._

Or you could give some examples, like: 
> _See [link](to-headset-lib) at `IVendorImplementation` for available properties/functions. A common use case would be: `if (sdk.headset.currentSelectedImplementation.isConnect) { /* do stuff */ }`